virt/virtual_machines/vm_networking/virt-attaching-vm-to-external-network.adoc

```diff
@@ -0,0 +1,80 @@
+:_content-type: ASSEMBLY
+[id="virt-attaching-vm-to-external-network"]
+= Attaching a virtual machine to an external network
+include::_attributes/common-attributes.adoc[]
+:context: virt-attaching-vm-to-external-network
+
+toc::[]
+
+{VirtProductName} is installed with a default internal `pod` network. If you want to attach VMs to an external network, however, you must create a Linux _network bridge_.
+
+Depending on the type of networks you are using, it may be necessary to update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration].
+
+Once a network bridge is created, you can start virtual machines on the external network in the same manner as you start virtual machines attached to internal networks, by using PXE booting, for example.
+
+= About configuring network bridges
+
+To attach virtual machines to additional networks, you must configure a Linux bridge.
+
+To attach a virtual machine to an additional network:
+
+. Configure a Linux network bridge.
+. Configure a bridge network attachment definition for a namespace in the web console or CLI.
++
+[NOTE]
+====
+The network attachment definition must be in the same namespace as the pod or virtual machine.
+====
+. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
+** In the web console, create a NIC for a new or existing virtual machine.
+** In the CLI, include the network information in the virtual machine configuration.
+
+[NOTE]
+====
+There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
+However, a network attachment definition provides a more efficient and more manageable configuration.
+====
+
+[id="virt-configuring-linux-bridge"]
+= Configuring a Linux bridge
+
+As a network administrator, you can create a Linux bridge interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
+
+include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+1]
+
+For more information about scheduling, interface types, and other node networking activities,
+see the xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking]
+section.
+
+[id="virt-creating-network-attachment-definition"]
+= Creating a network attachment definition
```

Comment:
This needs to be a level 2 header (==)

Reply:
Again this is the old file. So, I'm a little confused. The current TOC renders like this in Atom:

![image](https://user-images.githubusercontent.com/48329401/165639109-d6c73d84-e5eb-4b72-89ae-0b67c800ce4b.png)

if this isn't correct though, let me know.